recipes-extended/xen/files/libxl-syslog.patch

```diff
@@ -287,12 +287,15 @@ PATCHES
  }
  
  int main(int argc, char **argv)
-@@ -419,6 +424,8 @@ int main(int argc, char **argv)
+@@ -419,6 +424,11 @@ int main(int argc, char **argv)
          xtl_flags |= XTL_STDIOSTREAM_SHOW_DATE | XTL_STDIOSTREAM_SHOW_PID;
      logger = xtl_createlogger_stdiostream(stderr, minmsglevel, xtl_flags);
      if (!logger) exit(EXIT_FAILURE);
 +    /* OpenXT: we use syslog, we could probably comment out the above. */
 +    openlog("xl", LOG_CONS, LOG_USER);
++    /* OpenXT: xl writes to stderr, which can be a dangling pipe.  We don't
++     * want that terminating the process, so ignore it. */
++    signal(SIGPIPE, SIG_IGN);
```

Comment:
Couldn't we just redirect stdout and stderr to /dev/null if everything is making it to syslog by default?

Reply:
Ack.